src/tests/ftest/telemetry/telemetry_pool_metrics.yaml

```diff
@@ -33,7 +33,6 @@ ior:
   max_duration: 10
   # 1MiB
   transfer_size: 1048576
-  dfs_chunk: 1048576
```

Comment:
If you leave this, you don't need this line in the test:
`self.ior_cmd.dfs_chunk.update(self.ior_cmd.transfer_size.value)`

Reply:
Oups, forgot to remove this 😊 

- [ ] Remove useless `dfs_chunk` from yaml file